packages/fether-react/src/Send/TxDetails/TxDetails.js

```diff
@@ -0,0 +1,104 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
```

Comment:
Can you put TxDetails folder inside TxForm folder? Because that component is only used in TxForm